app/src/main/java/com/spoony/spoony/presentation/main/MainScreen.kt

```diff
@@ -111,7 +111,7 @@ fun MainScreen(
 
                 registerNavGraph(
                     paddingValues = paddingValues,
-                    navigateToExplore = navigator::navigateToExplore
+                    navigateToExplore = { navigator.navigateToExplore(fromRegister = true) }
```

Comment:
```suggestion
                    navigateToExplore = { navigator.navigate(MainTab.EXPLORE) }
``` 
P1: 이렇게 수정하고 navigateToExplore 함수를 변경하지 않는건 어떨까요?